packages/agoric-cli/lib/helpers.js

```diff
@@ -0,0 +1,39 @@
+export const makePspawn = ({
+  pspawnEnv = process.env,
+  log = console,
+  chalk,
+  spawn,
+}) => (cmd, cargs, { stdio = 'inherit', env = pspawnEnv, ...rest } = {}) => {
+  const color = (method, ...args) => {
+    if (chalk && chalk[method]) {
+      return chalk[method](...args);
+    }
+    return args.join(' ');
+  };
+
+  log.log(color('blueBright', cmd, ...cargs));
+  const cp = spawn(cmd, cargs, { stdio, env, ...rest });
+  const pr = new Promise((resolve, _reject) => {
+    cp.on('exit', resolve);
+    cp.on('error', e => {
+      let reason;
+      switch (e && e.code) {
+        case 'EACCES': {
+          reason = 'Access denied';
+          break;
+        }
+        case 'ENOENT': {
+          reason = 'File not found';
+          break;
+        }
+        default: {
+          reason = e;
+        }
+      }
+      log.error(color('yellow', `cannot execute ${cmd}:`), reason);
+      resolve(-1);
+    });
+  });
+  pr.cp = cp;
```

Comment:
`cp` looks like unused debug API. Otherwise, it might bear a more descriptive name.

Reply:
Thanks.  It is used in a few places.  I'll make it `childProcess`.